src/devtools/client/debugger/src/components/TestInfo/TestSteps.tsx

```diff
@@ -29,26 +29,26 @@ function useGetTestSections(
           acc[0].push({
             ...step,
             annotations: {
```

Comment:
:hand: Three thoughts here:

- This is a good candidate for a `useMemo()`
- Personally I'd like to see it as an object with named fields like `beforeEach` instead of `acc[0]` :)
- We probably don't have _tons_ of annotations, so `O(n)` runtime likely isn't a big deal, but we could consider normalizing each of those annotations arrays into a lookup table keyed by ID to avoid the repeated `.find() calls